en/website.json

```diff
@@ -720,5 +720,14 @@
     "shuffle_description": "Shuffle the current queue!",
     "resume_description": "Resumes the currently paused track.",
     "protection_bots_warning": "These limits might not apply to bots due to how they work, be careful from inviting unknown bots.",
-    "bot_require_invite": "SELECT THE BOT YOU WANT TO INVITE"
+    "bot_require_invite": "SELECT THE BOT YOU WANT TO INVITE",
+    "temp_module": "Temp Channels",
+    "category": "CATEGORY",
+    "temp_category_description": "New temporary channels will be under this category",
+    "temp_create_channel": "CREATE CHANNEL",
+    "temp_create_channel_description": "When a user joins this channel it creates a temp channel for them",
```

Comment:
"temp_create_channel_description": "When a user joins this channel, it creates a temporary channel for them.",